changelog/@unreleased/pr-2950.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Prefer Optional::orElseThrow over Optional::get
```

Comment:
Open question: Do we also want a more generic `PreferOptionalOrElseThrow` check to rewrite `Optional::get` & `opt.get()` to `Optional::orElseThrow` & `opt.orElseThrow()`

Reply:
I'd be in favor of this.